frontend/src/routes/Search.tsx

```diff
@@ -15,41 +22,84 @@ import { SmallDescription } from "../ui/metadata";
 import { Card } from "../ui/Card";
 import { Breadcrumbs, BreadcrumbsContainer, BreadcrumbSeparator } from "../ui/Breadcrumbs";
 import { MissingRealmName } from "./util";
-import { ellipsisOverflowCss } from "../ui";
+import { ellipsisOverflowCss, focusStyle } from "../ui";
 import { COLORS } from "../color";
 import { BREAKPOINT_MEDIUM, BREAKPOINT_SMALL } from "../GlobalStyle";
-import { keyOfId } from "../util";
+import { isExperimentalFlagSet, keyOfId } from "../util";
+import { Button } from "../ui/Button";
 import { DirectVideoRoute, VideoRoute } from "./Video";
 
 
 export const isSearchActive = (): boolean => document.location.pathname === "/~search";
 
+export type Item = Exclude<ItemType, "%future added value">;
```

Comment:
I think the name `Item` can be improved. I would expect `Item` to be the full item with all fields. `ItemType` is a fitting name. So maybe just import `ItemType as RawItemType`?